src/bef2c.c

```diff
@@ -416,8 +412,8 @@ int main (argc, argv)
           }
         }
       }
-      fclose (fi);
-      fclose (fo);
+      if (fi) fclose (fi);
+      if (fo) fclose (fo);
       if ((fi = fopen ("temp.c", "r")) != NULL)
```

Comment:
Trying to understand this part, I read the code and realized it defines two filehandle variables and then re-uses them for everything. Wow. This sort of coding probably seemed less odd back then than it does today.